### PR TITLE
Refactor `account.php`

### PR DESCRIPTION
**Pull request by everly-gif**
## What does this PR do?

Refactors `account.php`

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by christyjacob4 — APPROVED** [review]
Looks good 👍 
Once the CNAME records are fixed, we can merge this

File: app/controllers/api/account.php [hunk 1]
```diff
@@ -55,14 +63,7 @@
...
     ->inject('events')
-    ->action(function ($userId, $email, $password, $name, $request, $response, $project, $dbForProject, $audits, $usage, $events) {
-        /** @var Appwrite\Utopia\Request $request */
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $project */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Appwrite\Event\Audit $audits */
-        /** @var Appwrite\Stats\Stats $usage */
-        /** @var Appwrite\Event\Event $events */
+    ->action(function (string $userId, string $email, string $password, ?string $name, Request $request, Response $response, Document $project, Database $dbForProject, Audit $audits, Stats $usage, Event $events) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:66]:
```suggestion
    ->action(function (string $userId, string $email, string $password, string $name, Request $request, Response $response, Document $project, Database $dbForProject, Audit $audits, Stats $usage, Event $events) {
```

File: app/controllers/api/account.php [hunk 2]
```diff
@@ -264,10 +257,7 @@
     ->inject('request')
     ->inject('response')
     ->inject('project')
-    ->action(function ($provider, $success, $failure, $scopes, $request, $response, $project) use ($oauthDefaultSuccess, $oauthDefaultFailure) {
-        /** @var Appwrite\Utopia\Request $request */
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $project */
+    ->action(function (string $provider, ?string $success, ?string $failure, ?array $scopes, Request $request, Response $response, Document $project) use ($oauthDefaultSuccess, $oauthDefaultFailure) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:260]:
```suggestion
    ->action(function (string $provider, string $success, string $failure, array $scopes, Request $request, Response $response, Document $project) use ($oauthDefaultSuccess, $oauthDefaultFailure) {
```

File: app/controllers/api/account.php [hunk 3]
```diff
@@ -1166,16 +1103,9 @@
...
     ->inject('usage')
-    ->action(function ($limit, $offset, $response, $user, $locale, $geodb, $dbForProject, $usage) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $project */
-        /** @var Utopia\Database\Document $user */
-        /** @var Utopia\Locale\Locale $locale */
-        /** @var MaxMind\Db\Reader $geodb */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Appwrite\Stats\Stats $usage */
+    ->action(function (?int $limit, ?int $offset, Response $response, Document $user, Locale $locale, Reader $geodb, Database $dbForProject, Stats $usage) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:1106]:
```suggestion
    ->action(function (int $limit, int $offset, Response $response, Document $user, Locale $locale, Reader $geodb, Database $dbForProject, Stats $usage) {
```

File: app/controllers/api/account.php [hunk 4]
```diff
@@ -1231,12 +1161,7 @@
     ->inject('locale')
     ->inject('dbForProject')
     ->inject('usage')
-    ->action(function ($sessionId, $response, $user, $locale, $dbForProject, $usage) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $user */
-        /** @var Utopia\Locale\Locale $locale */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Appwrite\Stats\Stats $usage */
+    ->action(function (string $sessionId, Response $response, Document $user, Locale $locale, Database $dbForProject, Stats $usage) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:1164]:
```suggestion
    ->action(function (?string $sessionId, Response $response, Document $user, Locale $locale, Database $dbForProject, Stats $usage) {
```

File: app/controllers/api/account.php [hunk 5]
```diff
@@ -1526,15 +1420,7 @@
...
-    ->action(function ($sessionId, $request, $response, $user, $dbForProject, $locale, $audits, $events, $usage) {
-        /** @var Appwrite\Utopia\Request $request */
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $user */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Utopia\Locale\Locale $locale */
-        /** @var Appwrite\Event\Audit $audits */
-        /** @var Appwrite\Event\Event $events */
-        /** @var Appwrite\Stats\Stats $usage */
+    ->action(function (string $sessionId, Request $request, Response $response, Document $user, Database $dbForProject, Locale $locale, Audit $audits, Event $events, Stats $usage) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:1423]:
```suggestion
    ->action(function (?string $sessionId, Request $request, Response $response, Document $user, Database $dbForProject, Locale $locale, Audit $audits, Event $events, Stats $usage) {
```

File: app/controllers/api/account.php [hunk 6]
```diff
@@ -1613,17 +1499,7 @@
...
-        /** @var boolean $force */
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $user */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Utopia\Database\Document $project */
-        /** @var Utopia\Locale\Locale $locale */
-        /** @var Appwrite\Event\Audit $audits */
-        /** @var Appwrite\Event\Event $events */
-        /** @var Appwrite\Stats\Stats $usage */
+    ->action(function (string $sessionId, Request $request, Response $response, Document $user, Database $dbForProject, Document $project, Locale $locale, Audit $audits, Event $events, Stats $usage) {
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:1502]:
```suggestion
    ->action(function (?string $sessionId, Request $request, Response $response, Document $user, Database $dbForProject, Document $project, Locale $locale, Audit $audits, Event $events, Stats $usage) {
```

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**